lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,12 @@ div.course-wrapper {
       }
     }
 
+    #sr-is-focusable,
```

Comment:
Can you give me the cliff notes on why this selector is an ID and not a class? What is it selecting?